esp-box/Kconfig

```diff
@@ -47,6 +47,12 @@ menu "Board Support Package"
         range 1 100
         help
             Period of LVGL tick timer.
+
+        config BSP_DISPLAY_LVGL_FS_SUPPORT
+            bool "LVGL file system support"
+            default n
+            help
+                Enable or disable file system support for LVGL.
```

Comment:
Do we really need this Kconfig option?

If yes, could we add it to `esp_lvgl_port`'s Kconfig?

Reply:
Remove this Kconfig option in this PR. It would be better to control this in the BSP layer, but not in the esp_lvgl_port layer.